Open Judge System/Workers/OJS.Workers.ExecutionStrategies/PhpProjectWithDbExecutionStrategy.cs

```diff
@@ -0,0 +1,127 @@
+﻿namespace OJS.Workers.ExecutionStrategies
+{
+    using System.Collections.Generic;
+    using System.IO;
+
+    using OJS.Common;
+    using OJS.Common.Extensions;
+    using OJS.Workers.Checkers;
+    using OJS.Workers.ExecutionStrategies.SqlStrategies.MySql;
+    using OJS.Workers.Executors;
+
+    public class PhpProjectWithDbExecutionStrategy : PhpProjectExecutionStrategy
+    {
+        protected const string DatabaseConfigurationFileName = "db.ini";
+        protected const string TestRunnerClassName = "JudgeTestRunner";
```

Comment:
will I find this const in more than one files if I search for it?

Reply:
No.